README.md

```diff
@@ -35,6 +36,56 @@ $ swaggerhub configure
 ```
 
 Environment variables take precedence over the configuration file created by this command.
+
+## Additional configuration for SwaggerHub On-Premise
+
+If your SwaggerHub On-Premise instance uses a **self-signed or privately signed SSL certificate**, there are additional steps required to make SwaggerHub CLI trust this certificate.
+
+By default, Node.js rejects self-signed or privately signed SSL certificates because their root CA is not known. You will see an error like this in CLI output:
```

Comment:
Similarly, you should use the definite article here also as we are referring to a specific noun - there is only one CLI.

`You will see an error like this in the CLI output.`